runtime/oti/j9nonbuilder.h

```diff
@@ -5690,6 +5742,12 @@ typedef struct J9JavaVM {
 	omrthread_monitor_t jclCacheMutex;
 	UDATA arrayletLeafSize;
 	UDATA arrayletLeafLogSize;
+	UDATA contiguousIndexableHeaderSize;
+	UDATA discontiguousIndexableHeaderSize;
+#if defined(J9VM_ENV_DATA64)
+	UDATA isIndexableDataAddrPresent;
+	BOOLEAN isIndexableDualHeaderShapeEnabled;
```

Comment:
Do we need both of these members?

Reply:
Yes, they are both needed since they serve different purposes. `isIndexableDualHeaderShapeEnabled` is required for the command line option in case we would like to disable the dual header shape functionality. I believe though down the road `isIndexableDualHeaderShapeEnabled` can be removed (maybe even shortly after this PR in the PR where dual-header functionality is enabled by default).